feature-5 folder/feature-5.txt

```diff
@@ -0,0 +1,5 @@
+feature-5 begin
+
+feature-5 content
```

Comment:
comment about content,
please add an exclamation mark

Reply:
Can I add a dot?

Reply:
Yes, you can add a dot.

Reply:
Please add a dot or exclamation mark

Reply:
the conversation is not resolved

Reply:
done